src/main/java/org/minions/devfund/damianvp/moviedatabase/Movie.java

```diff
@@ -0,0 +1,92 @@
+package org.minions.devfund.damianvp.moviedatabase;
+
+import java.util.ArrayList;
+
+/**
+ * Movie class.
+ */
+public class Movie {
+
+    private String name;
+    private ArrayList<Actor> actors;
+    private double rating;
+
+    /**
+     * Constructor.
+     * @param name String type.
+     */
+    Movie(final String name) {
+        this.name = name;
+        this.actors = new ArrayList<>();
+        this.rating = 0.0;
```

Comment:
this is redundant, consider removing it.
double attributes of class are 0.0 by default

Reply:
sorry, I forget remove this